index.html

```diff
@@ -575,11 +761,18 @@ <h4>Derek Yen</h4>
           <p class="text-muted">UROP<br/>Gender/Novels<br>Comp/Hist</p>
         </div>
       </div>
+      <div class="col-6 col-sm-4 col-md-2">
```

Comment:
we don't list people who didn't complete at least half a term as members, so Jawad should be removed.